.travis.yml

```diff
@@ -0,0 +1,53 @@
+dist: xenial
+
+language: python
+
+python:
+  - "3.7"
+
+services: docker
+
+env:
+  - COMPOSE_PROJECT_NAME: 'ford3'
+    COMPOSE_FILE: 'docker-compose.yml:docker-compose.override.yml'
+
+before_install:
+  # Attempt to detect docker context changes
+  - deployment/scripts/travis/check-docker-context.sh
+  - pushd deployment
+  - source compose.env.sh
+  - cat docker-compose.override.yml
+  - popd
+
+install:
+  - pip install -U docker-compose
+  - pip install -r REQUIREMENTS-dev.txt
+
+script: skip
+
+jobs:
+  include:
+    - stage: Test
+      script:
+        - pushd deployment
+        - echo "Preparing Docker Stack"
+        - make build up wait-for-db migrate collectstatic
+        - make status
+        - echo "Docker stack ready"
+        - echo "Preparing unittests"
+        - make flake8
+        - echo "Run unittests"
```

Comment:
`make test` is already present

Reply:
Yes, I saw that you create another Make command, so I didn't make a new one here to avoid conflict.